information about vscode extension search functionality is added (#660)
playcanvas/developer.playcanvas.com: docs/user-manual/scripting/vscode-extension.md

* **Enhanced Code Editing**: Leverage the full power of VS Code's editing features, such as IntelliSense, code snippets, GitHub Copilot and advanced refactoring tools.
* **Source Control**: Switch branches to work in isolation.
* **Collaboration**: Collaborate on scripts and shaders with your team, with changes reflected immediately in the PlayCanvas editor. Use the `Pull Latest` command from the context menu to pull latest changes into your VS Code environment.  

> * **Find In Files**: The extension supports searching in project files - use `PlayCanvas: Search` command to search in a project or `PlayCanvas:Find In Folder` from the context menu to search in a folder.

Reviewer: ```suggestion
* **Find In Files**: The extension supports searching in project files - use the `PlayCanvas: Search` command to search in a project or `PlayCanvas:Find In Folder` from the context menu to search in a folder.
```

### Configuration

> * [Generate a new API Key](/user-manual/api/#authorization) and copy a new authentication token to clipboard.
> * Open your project in VS Code by opening the Command Pallette (`CTRL`+`Shift`+`P`) and typing "PlayCanvas: Add Project".
> * Enter new token in an input box (only once).

Reviewer: ```suggestion
* [Generate a new API Key](/user-manual/api/#authorization) and copy it to the clipboard.
* Open your project in VS Code by opening the Command Palette (`CTRL`+`Shift`+`P`) and typing "PlayCanvas: Add Project".
* Paste your API token into the input box. This only needs to be done once.
```
